### PR TITLE
Add API documentation for staging projects

### PR DESCRIPTION
**Pull request by dmarcoux**

### CODE REVIEW

File: docs/api/api/api.txt [hunk 1]
```diff
@@ -1933,6 +1933,19 @@ XmlResult: required_checks
 == Staging
   WARNING: This API is currently in beta and unstable
 
+=== Staging Projects
+
+<staging_workflow_project>: The name of the staging workflow project, e.g. home:user1
```
Comment (Ana06) [inline docs/api/api/api.txt]: what about taking a more realistic name than `home:user1`? like Factory e.g.

File: src/api/app/views/staging/staging_projects/_broken_packages.xml.builder [hunk 2]
```diff
@@ -1,4 +1,4 @@
-builder.broken_package(count: count) do |broken_package|
+builder.broken_packages(count: count) do |broken_package|
```
Comment (Ana06) [inline src/api/app/views/staging/staging_projects/_broken_packages.xml.builder]: you have to change it in the tests as well :pray:

**Comment by dmarcoux** [comment]
@Ana06 Please have another look

**Comment by dmarcoux** [comment]
[Provide API endpoint to query overall state of the staging project](https://trello.com/c/hHP98jTx/1014-provide-api-endpoint-to-query-overall-state-of-the-staging-project)

**Pull request merged**